com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -699,11 +720,12 @@ static float GetMaxBokehRadiusInPixels(float viewportHeight)
             return Mathf.Min(0.05f, kRadiusInPixels / viewportHeight);
         }
 
-        void DoBokehDepthOfField(CommandBuffer cmd, int source, int destination, Rect pixelRect)
+        void DoBokehDepthOfField(Camera camera, CommandBuffer cmd, int source, int destination, Rect pixelRect)
```

Comment:
Do we need to add the camera as a parameter?

Reply:
Fixed